packages/data-lake-api/src/__tests__/utilities/getTestDataLakeDatabase.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { requireEnv, getEnvVarOrDefault } from '@tupaia/utils';
+import { DataLakeDatabase } from '../../DataLakeDatabase';
+
+let readDatabase: DataLakeDatabase | null = null;
+let writeDatabase: DataLakeDatabase | null = null;
+
+export function getTestDatabase() {
+  if (!readDatabase) {
+    readDatabase = new DataLakeDatabase();
```

Comment:
```suggestion
export function getTestDatabase() {
  if (!readDatabase) {
    readDatabase = new DataLakeDatabase(getConnectionConfig());
```

Can we move `getConnectionConfig()` here? Just wanna make these functions be more consistent.

Reply:
Sure thing 👍